5.5/Dockerfile

```diff
@@ -20,10 +20,23 @@ LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database ser
       io.openshift.expose-services="3306:mysql" \
       io.openshift.tags="database,mysql,mysql55"
 
+# Support for atomic run/install/uninstall
+LABEL INSTALL="docker run -t -i --rm --privileged -u 0:0 -v /:/host --net=host \
```

Comment:
`$OPTx` is not only for environment variables, but for modifying the command overall. Let's say you have the `--verbose` switch which is off by default, but you can turn it on if there is `$OPTx` on the right place.

Images should work without these options being used, but they should let you tweak the behaviour of image in special cases (like debuging as I described above)